configs/opencv-ovms/scripts/run_instance_segmentation.sh

```diff
@@ -6,7 +6,7 @@
 #
 
 rmDocker=--rm
-if [ ! -z "$DEBUG" ]
+if [ ! -n "$DEBUG" ]
```

Comment:
should be just -n  no !  as it is meant to check non-empty $DEBUG env
```suggestion
if [ -n "$DEBUG" ]
```